From acfc63f6f42ccc8917652def2fcd330256f6e2fd Mon Sep 17 00:00:00 2001
From: Jan Jasper de Kroon <jajadekroon@gmail.com>
Date: Sat, 26 Nov 2022 20:33:28 +0100
Subject: [PATCH] revert:  kbuild: fix -Wimplicit-function-declaration in
 license_is_gpl_compatible

This commit introduces the include <linux/string.h> into licences.h,
which seems to have been introduced for building with Clang,
but breaks building with gcc. Reverting this will fix the build proces.
---
 include/linux/license.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/linux/license.h b/include/linux/license.h
index ad937f57f..7cce390f1 100644
--- a/include/linux/license.h
+++ b/include/linux/license.h
@@ -2,8 +2,6 @@
 #ifndef __LICENSE_H
 #define __LICENSE_H
 
-#include <linux/string.h>
-
 static inline int license_is_gpl_compatible(const char *license)
 {
 	return (strcmp(license, "GPL") == 0
-- 
2.34.3

